docs/js/tiles.js

```diff
@@ -149,6 +149,9 @@ function animateDiagram(container, forceCenterX, forceCenterY) {
     }
   }
 }
+function setInkscapTemplate(linkNode) {
+  linkNode.href = 'data:text/plain,' + InkscapeTemplate.fromUrl(submitQuery())
```

Comment:
The expression produces tab and newline characters, but assigned to the href, they get lost.